Scripts/DCS-BIOS/lib/common/Logger.lua

```diff
@@ -146,7 +146,6 @@ end
 -- Break ALL when max depth reached
 -- Break when data logged exceeds limit
 
---- @private
```

Comment:
Why was this removed?

Reply:
I tried calling it directly when I wasn't getting any output. Will change.

Reply:
Ah the price annotation just affects intellisense for the lua language server plugin (and it will yell at you if you try to use it outside of the defined scope) but it doesn't affect runtime behavior